### DISCUSSION TITLE
Does not have Brazil for the country field in SMS 2FA

### DISCUSSION THREAD

**Discussion by gabriel-secchi** [community/community#129830] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Olá.
Estou tentando configurar o SMS 2FA para minha conta, contudo não está disponível para o campo "País ou Código da região" o meu país.
Deveria ter: Brasil +55

Teria como adicionar para que eu possa aplicar essa método de segurança?

**Comment by ebndev** [comment]
Hey there @gabriel-secchi,

Thanks so much for reaching out in GitHub's Community! If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).
